e2e/test/scenarios/dashboard-cards/click-behavior.cy.spec.js

```diff
@@ -694,13 +695,14 @@ describe("scenarios > dashboard > dashboard cards > click behavior", () => {
       cy.intercept("GET", "/api/collection", cy.spy().as("collections"));
 
       clickLineChartPoint();
-      cy.location().should(({ hash, pathname }) => {
```

Comment:
Previously this incorrectly created an adhoc question. Now there is a direct link by id.